_schemaV2.graphql

```diff
@@ -1607,6 +1607,13 @@ enum ArtworkAggregation {
   TOTAL
 }
 
+enum ArtworkAttributionClassType {
```

Comment:
Is `ConsignmentAttributionClass` referring to the same thing? Just wondering if we could unify it into only one enum.

Reply:
That would be great but I guess `ConsignmentAttributionClass` is not an input type, right?